apps/platform-app/src/app/[locale]/(profile)/(components)/CVuploader/CvUploader.module.scss

```diff
@@ -0,0 +1,67 @@
+.hidden {
+    display: none;
+  }
+  
+  .container {
+    position: relative;
+    margin: 30px 0 5px 0;
+    display: flex;
+    align-items: center;
+    button {
+      color: #a687ff;
+      font-size: 14px;
+      font-style: normal;
+      font-weight: 500;
+      line-height: 20px;
+      cursor: pointer;
+    }
+    label[for='file-input'] {
+      cursor: pointer;
+    }
+    @include mobile {
```

Comment:
**suggestion**: Consider adding a comment to explain the purpose of the '@include mobile' mixin for better maintainability.